### ISSUE TITLE
Does not replace Story Text using -deh command.

### ISSUE THREAD

**Issue by hackneyed-one** [fabiangreffrath/woof#189]
Version of Woof: Compiled from source. Version 5.0.0 from commit caf37cb on May 4, 2021

OS: Linux. Ubuntu 20.04.2 LTS.

Setup: New config file. For a quick test from console I run "./woof -config test.cfg -deh AV.DEH -warp 06" Then idclip, go to the left and press the exit switch.

Bug: When loading a .deh file from the command line Level Names are replaced but Story Text is NOT. I've only checked after MAP06 and MAP30.

I also tested btsx_e1.deh with the same results. However if I only load the wad files thus using the embedded dehacked or I extract the DEHACKED.txt lump and load it from the command line "./woof -config test.cfg -deh DEHACKED.txt -warp 06" it works. Opening DEHACKED.txt and btsx_e1.deh in a text editor I see they are structured vary differently. btsx_e1.deh being very similar to AV.DEH.

Expected: Story Text to be replaced when loading any vanilla compatible deh file from the command line that replaces Story Text. Like Chocolate Doom and Crispy Doom work for me.

Thanks!

**Comment by fabiangreffrath** [comment]
Hm, I guess this might have to do with CRLF line endings.

**Comment by fabiangreffrath** [comment]
Just checked with AV.deh and the story text after MAP06 was replaced properly.

**Comment by fabiangreffrath** [comment]
The DEHACKED lump embedded into btsx_e1a.wad is in BEX format whereas the separate btsx_e1.eh is in standard Vanilla DEH format. However, both are supposed to achieve the same. So, they shouldn't be loaded together or else they will overwrite each others string replacements.

**Comment by hackneyed-one** [comment]
I saw you just made a Window fix so on a hunch I downloaded the pre compiled win32 release of 5.0.0... and it works in wine! I run "wine woof.exe -deh AV.DEH" from my bash console and story text is working there.

So it seems to be a Linux native problem for Woof. I don't know if something is breaking in my compile or what. I haven't edited the source. I can check the output or make a debug release but I don't really know what to look for or do. 

Lastly, don't worry I'm not loading them at the same time thanks.
